package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mafmt",
-  "version": "3.0.0",
+  "version": "3.0.1",
```

Comment:
@mkg20001 don't update this on PR. This gets updated on release by `aegir`

Reply:
@diasdavid Updated it